### ISSUE TITLE
NTR, pulmo: traction bronchiolectasis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5584]
**Preferred term label:**
traction bronchiolectasis

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
CT findings represent irregular bronchiolar dilatation caused by surrounding retractile pulmonary fibrosis. Dilated airways are usually identifiable as such but may be seen as microcysts. 

PMID: 18195376

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sarcoidosis, ORPHA:797
Idiopathic pulmonary fibrosis (ORPHA:2032, OMIM 178500)
Hypersensitivity pneumonitis (ORPHA:31740, OMIM 145300)

**Your nano-attribution (ORCID)**
LMU: crapp, @mgriese, @katrinknoflach
